config/routes.rb

```diff
@@ -4,7 +4,7 @@
 
   root controller: :laundry_rooms, action: :index
   resources :laundry_rooms, only: [:index] do
-    get :create_booking
+    get :create_booking, :delete_booking
```

Comment:
This feels like a weird way of setting a route. Where did you get it from?

Reply:
From this syntax in Rails Routing in Rails Guide `resources :photos, :books, :videos`
Do you mean that the problem is that it says `get`and not `resources`? I don't know what the right way to set a route is.

Reply:
I'll just move it down to its own line, I guess. `get :delete_booking`